pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/route/KitUploadRoute.java

```diff
@@ -289,17 +288,15 @@ private void uploadKit(@NonNull DDPInstance ddpInstance, @NonNull KitType kitTyp
         }
     }
 
-    private boolean checkAndSetParticipantIdIfKitExists(DDPInstance ddpInstance, Connection conn, KitRequest kit, String participantGuid,
-                                                        String participantLegacyAltPid, int kitTypeId) {
-        boolean isKitExsist = false;
-        if (checkIfKitAlreadyExists(conn, participantGuid, ddpInstance.getDdpInstanceId(), kitTypeId)) {
-            isKitExsist = true;
-            kit.setParticipantId(participantGuid);
-        } else if (checkIfKitAlreadyExists(conn, participantLegacyAltPid, ddpInstance.getDdpInstanceId(), kitTypeId)) {
-            isKitExsist = true;
-            kit.setParticipantId(participantLegacyAltPid);
+    private boolean checkAndSetParticipantIdIfKitExists(DDPInstance ddpInstance, Connection conn, String bspCollaboratorParticipantId,
+                                                        int kitTypeId) {
+        if (checkIfKitAlreadyExists(conn, bspCollaboratorParticipantId, ddpInstance.getDdpInstanceId(), kitTypeId)) {
+            return true;
         }
-        return isKitExsist;
+        if (checkIfKitAlreadyExists(conn, bspCollaboratorParticipantId, ddpInstance.getDdpInstanceId(), kitTypeId)) {
```

Comment:
what is the difference between this and L293?

Reply:
😂